docs/DeveloperGuide.adoc

```diff
@@ -447,6 +447,73 @@ In other words, when RestaurantBook#deleteMember() or RestaurantBook#deleteIngre
 
 * Decision: Alternative 1 was adopted as it is an easier implementation and it makes it more convenient for the user to delete a record. It is intuitive that if a deleteMember or deleteIngredient command is called and these objects cease to exist in the restaurant book, the corresponding Booking or Recipe containing these objects should also cease to exist. To deal with the possibility that the member or ingredient is accidentally deleted, and all referenced bookings and recipes is unintentionally deleted, an additional field in the delete command can be added as a confirmation that these records should be deleted.
 
+=== The `stats` command
+
+When the `stats` command is entered, the current statictis of the amount of bookings is to be displayed to the user in the form of a bar chart.
+
+==== Current implementation
+
+The `RestaurantBookParser` class would detect if the user has keyed in the keyword `stats`. If it does, `RestaurantBookParser` would call the `Statistics` class. The `Statistics` class is used to determine what data is to be displayed to the user in the bar chart (ie what are the axes, values, and legends of the bar chart). It would call the `StatsWindow` class which handles the UI. The bar chart will appear in a new window and does not affect the current window.
+
+==== Design Considerations
+
+===== Aspect: Implementation of data structure
+
+* **Alternative 1 (current choice):** Doing the calculation only when the user wants to view the statistics.
+** Pros: Calculating only when needed is more efficient.
+** Cons: The command would take some time to calculate before being able to display the statistics.
+
+* **Alternative 2:** Every change to the database updates the statistics on the fly.
+** Pros: Displaying the statistics on the UI would be faster.
+** Cons: Need to change the implementation of every command that causes a change to the database. Calculations have to be done every operation, affecting efficiency.
```

Comment:
Conflicting ideas of efficiency? So which is more efficient? Potentially confusing.